test/package.json

```diff
@@ -31,8 +33,8 @@
 		"node": ">=10"
 	},
 	"scripts": {
-		"test:run": "wdio ./suites/$SUITE/$SUITE.conf.js",
-		"test:run-filter": "wdio ./suites/$SUITE/$SUITE.conf.js --spec $FILTER",
+		"test:run": "wdio ./suites/$SUITE/$SUITE.conf.ts",
+		"test:run-filter": "wdio ./suites/$SUITE/$SUITE.conf.ts --spec $FILTER",
```

Comment:
Turns out you can fully just run these with Typescript without explicitly compiling them to Javascript first